pkg/cli/gen.go

```diff
@@ -239,7 +240,18 @@ Output the list of cluster settings known to this binary.
 					defaultVal = override
 				}
 			}
-			row := []string{wrapCode(name), typ, wrapCode(defaultVal), setting.Description()}
+
+			settingDesc := setting.Description()
+			if strings.Contains(name, "sql.defaults") {
+				settingDesc = fmt.Sprintf(`%s
+This cluster setting is being kept to preserve backwards-compatibility.
+This session variable default can now be configured using ALTER ROLE: %s`,
+					setting.Description(),
+					docs.URL("alter-role.html"),
```

Comment:
probably have to give the correct URL here; `docs.URL` automatically sets this to `/docs/<version_number>` by default, but since we're autogenerated a file it uses `/docs/dev` which is probably not what we want. we should probably use `/docs/stable` for now.